src/main/kotlin/com/bugsnag/android/gradle/BugsnagReleasesTask.kt

```diff
@@ -60,20 +57,15 @@ sealed class BugsnagReleasesTask(
         description = "Assembles information about the build that will be sent to the releases API"
     }
 
+    @get:Input
+    override val manifestInfoFile: Property<AndroidManifestInfo> = objects.property()
```

Comment:
nit: might be worth renaming the property names across these tasks as this is no longer a File